### DISCUSSION TITLE
Parallel vs SPI

### DISCUSSION THREAD

**Discussion by unregistered33** [Bodmer/TFT_eSPI#1555] [Q&A - General]
Not issue, just question. What works faster: SPI or Parallel?

**Comment by Bodmer** [comment]
Typically 8 bit parallel is faster unless you can take advantage of DMA, where SPI can win. It is also dependant on the processor, the ESP32 can only bitbash an 8 bit parallel bus at 10Mbytes/s. Other processors such as RP2040 or STM32 can output data at a rate higher than the display will accept.

If the display has a SPI interface that can be clocked fast (>50MHz) then there will not be so much difference.

**Comment by unregistered33** [comment]
Thanks!
